cypress/e2e/11-support-manage-facility.cy.js

```diff
@@ -38,11 +40,25 @@ describe("Support admin: manage facility", () => {
 
     // selects organization
     cy.wait("@GetAllOrganizations");
-    cy.get("div.bg-base-lightest select").first().select(organizationId);
 
-    // selects facility
+    // selects org combo box
+    cy.get("[data-testid=\"org-selection-container\"] " +
+      "> [data-testid=\"combo-box\"] ")
+      .within(() => {
+        // within the org selection box
+        cy.get("[data-testid=\"combo-box-input\"]").click();
```

Comment:
I thought it was a little gross to use content-unaware selectors in our tests like this but couldn't get the selectors to work because of the way `select` within the combo box is hidden from view on default (as Cypress complains about the `select` not being in view if you try to select it). If anyone has suggestions on improving the way we're choosing selectors, would love that!

Reply:
for this the other alternative I could think of if you had both the org name and facility name handy is...
`cy.get('input[role="combobox"]').first().type("${ORG_NAME}{enter}");`
`cy.get('input[role="combobox"]').last().type("${FACILITY_NAME}{enter}");`

Reply:
thanks for the suggestions elisa! added.